### ISSUE TITLE
Breakpoint doesn't stop (VSCode1.4/Ubuntu16.04/RN0.31)

### ISSUE THREAD

**Issue by sam-n-johnston** [Microsoft/vscode-react-native#280]
## Actual Behavior
1. `react-native init test`
2. Create .babelrc with: 

  ```
  {
    "presets": [
      "react-native" // this is required for debugging with react-native/packager/transformer
    ],
    "plugins": [],
    "sourceMaps": true // must be true react-native/packager/transformer using with node-module-debug
    // because of some bugs from vscode-node-debug & vscode-react-native, "sourceMaps" cannot be "inline" or "both"
  }
  ```
3. Create breakpoint on `return` line of render method in index.android.js  
4. Create and run Debug Android from VS Code
5. Runs the app without stopping.

## Expected Behavior
1. Expect to stop at return statement.

## Software versions
- React-Native VS Code extension version: 
- VSCode version: 1.4
- OS platform and version: Ubuntu 16.04
- React Native version: 0.31.0

I also added a `componentWillMount` function to the `Component` with a few `console.log("test")` statements and breakpoints. The `console.log()` works fine, but VS Code still doesn't stop at the breakpoint. 

This works fine on my Windows machine, but it has a different version of VSCode and React Native. I'm not quite sure what the problem could be. 

When I put a `debugger;` statement, it stops in the ./.vscode/.react/index.android.bundle file at the `debugger` line and I can continue with F10 as expected, but it isn't the original code, so it can be quite hard to follow.

**Comment by MSLaguana** [comment]
Are you using the currently published version of the extension, or did you build from sources? React-native's sourcemaps use a feature that we didn't initially support, and so we had to make a change to the extension which hasn't yet been published to the vscode marketplace.

**Comment by sam-n-johnston** [comment]
I am using the currently published version of the extension. 

So I need to build the extension from source to get this to work?

**Comment by MSLaguana** [comment]
That may help, although I also see that it worked for you on a windows machine using different versions of vscode and react-native (but presumably still not building our extension yourself).

What version of vscode and react-native are you using there?

**Comment by sam-n-johnston** [comment]
On my windows I have:
-React-Native VS Code extension version: not sure how to find this without the new extension button
-VSCode: 1.2.1
-OS platform and version: Windows 10
-React Native version: 0.26.2

I don't even have `"sourceMaps": true` in my `.babelrc` file on windows. It works just by installing the extension out of the box.

**Comment by sam-n-johnston** [comment]
I also tested on another version of Ubuntu I have on another computer and it works perfectly. 

- React-Native VS Code extension version: Not sure...  
- VSCode: 1.3.1
- OS platform and version: Ubuntu 14.04
- React Native version: 0.31

**Comment by MSLaguana** [comment]
What version of nodejs do you have on these various machines? We currently don't debug properly if you are using nodejs 6.X, could that be the issue you are hitting?

**Comment by mbraude** [comment]
@sam-n-johnston we've tried reproing this several times but haven't been able to. I am going to close this. If you still have trouble, please reactivate and let us know so we can take another look.

**Comment by JoseFMP** [comment]
For me this issue persists.

**Comment by ruslan-bikkinin** [comment]
Hi @JoseFMP, please submit separate issue, describing you specific case.
